doc/data-formats.md

```diff
@@ -136,6 +149,11 @@ Consensus mutation files are products of the [`analyses/snv-callers`](https://gi
   * `pbta-snv-consensus-mutation-tmb-all.tsv` includes tumor mutation burden statistics that are calculated based calculated from Strelka2 and Mutect2 SNV consensus, and the intersection of Strelka2 and Mutect2 BED windows sizes.
   * `pbta-snv-consensus-mutation-tmb-coding.tsv` contains coding only tumor mutation burden statistics calculated from the number of coding sequence Strelka2, Mutect2, and Lancet consensus SNVs and size of the intersection of all three callers' BED windows and the Gencode v27 coding sequence regions. 
 
+### Consensus Copy Number File
```

Comment:
Can we group this with the GISTIC information below (line 190) please?

Reply:
updated